### ISSUE TITLE
Can't resize QR code svg

### ISSUE THREAD

**Issue by MartinSStewart** [pablohirafuji/elm-qrcode#9]
The SVG generated with `toSvg` doesn't seem to provide any way to choose a size for the QR code. 

The documentation says 
> **Tip**: You can determine the size of the generated svg setting
`width` and `height` styles.

but the only way I know of doing this is by adding a CSS rule such as 
```css
svg {
  width: 500px;
  height: 500px;
}
```
which can potentially affect other SVG content.

Is there something I'm missing here?

**Comment by pablohirafuji** [comment]
You are correct.

I recommend wrapping your `svg` in a `div` with a `class`, so you can target the `svg` with the parent class:

```
.parentClass svg {
     width: 500px;
     height: 500px;
}
```

It does add an extra `div`, but get the job done.

I will keep this use case in mind in the next API iteration, thanks.

**Comment by MartinSStewart** [comment]
Thanks, I didn't know how to do that with CSS. 

In the future, maybe there could be a `toStyleWith` that takes a `List Html.Attribute` as an additional parameter? The generated SVG would only set the viewbox and leave everything else up to the provided html attributes.

**Comment by pablohirafuji** [comment]
> Thanks, I didn't know how to do that with CSS.

No problem.

> In the future, maybe there could be a toStyleWith that takes a List Html.Attribute as an additional parameter?

Yes, probably something like that. I'm still gathering feedbacks and grouping all changes I want to make.

**Comment by pablohirafuji** [comment]
Fixed in [4.0.0](https://package.elm-lang.org/packages/pablohirafuji/elm-qrcode/latest/).
